releasenotes/notes/ddsketch-api-a84ffc0875bbacd6.yaml

```diff
@@ -0,0 +1,7 @@
+---
+prelude: >
+  The public API
```

Comment:
how does this get rendered?

I don't think this gets associated with this upgrade note it just gets added at the top?

Reply:
Oh wait, that totally shouldn't be there lol.

Reply:
Should be fixed now